targets/TARGET_NXP/TARGET_LPC176X/reset_reason.c

```diff
@@ -0,0 +1,62 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2017 ARM Limited
```

Comment:
New files, 2019 year. Can you also use SPDX idenfier? Check `platform` files  for the proper license

Reply:
Done